login_server/skeema/gradido_login/insert/setup_hedera_group.sql

```diff
@@ -1,13 +1,13 @@
-INSERT INTO `groups` (`id`, `alias`, `name`, `url`, `home`, `description`) VALUES
-(1, 'dockerStage2', 'docker stage2 gradido group', 'localhost', '/', 'gradido test group for docker and stage2');
+INSERT INTO `groups` (`id`, `alias`, `name`, `url`, `host`, `home`, `description`) VALUES
+(2, 'dockerStage2', 'docker stage2 gradido group', 'localhost', 'nginx', '/', 'gradido test group for docker and stage2');
 
 INSERT INTO `hedera_ids` (`id`, `shardNum`, `realmNum`, `num`) VALUES
 (1, 0, 0, 3),
 (2, 0, 0, 3327),
 (3, 0, 0, 413151);
 
 INSERT INTO `node_servers` (`id`, `url`, `port`, `group_id`, `server_type`, `node_hedera_id`, `last_live_sign`) VALUES
-(1, 'http://0.testnet.hedera.com', 50211, 0, 4, 1, '2000-01-01 00:00:00');
+(1, 'http://0.testnet.hedera.com', 50211, 0, 4, 2, '2000-01-01 00:00:00');

```

Comment:
This should cause duplicate keys exception when applying assuming that `id` is a primary key. MySQL allows `INSERT REPLACE`, but I am unsure wether thats the correct approach.

While on live this data is not present yet, the other installations may run into problems (e.g. dockers build lately, maybe staging)

Reply:
Maybe I should move the insert scripts into another folder to prevent confusion. 
The insert scripts cannot be run by skeema. 
They only be run by mariadb container if db_vol volume is empty also if you have a empty db.

Reply:
I have moved the insert scripts.